installing/installing_azure/installing-azure-customizations.adoc

```diff
@@ -26,6 +26,11 @@ include::modules/ssh-agent-using.adoc[leveloffset=+1]
 
 include::modules/installation-azure-marketplace-subscribe.adoc[leveloffset=+1]
 
+[role="_additional-resources"]
```

Comment:
@dfitzmau This xref is out of context. This is an IPI assembly. You are cross referencing content for a UPI install. Please remove.

Reply:
OK. 

I'm making changes to a module (`installation-azure-marketplace-subscribe`) in the assembly `installing-azure-user-infra.adoc . I have seen many references to UPI in this assembly. Just want to double-check with you that this assembly covers IPI only? 

My intention was to steer the user to a complete file structure without having to possibly duplicate the effort.

Reply:
Yes. This file is an IPI installation topic. The xref is not required.